### ISSUE TITLE
the performance is not increasing during supervised training step

### ISSUE THREAD

**Issue by cszmli** [snakeztc/NeuralDialog-LaRL#1]
Hi Tiancheng, I run `python experiments_woz/sl_cat.py` to pretrain the model with supervised learning. I found the performance was not changing according to the metrics although the valid loss decreased (from 1.9 to 1.5).  This is my machine environment:

- `python 3.6`
- `pytorch 0.4.0` (I also tried pytorch 1.1)
- cuda 9.0
- `cudnn 7.1`

Did I miss something in the code or the parameters? I attached a screenshot from the training step. 

![image](https://user-images.githubusercontent.com/18543171/58277008-a07cd480-7d4d-11e9-8215-cd2f9b783b96.png)

**Comment by snakeztc** [comment]
Thanks for the issue. I am looking into this now.

**Comment by snakeztc** [comment]
@cszmli this should be fixed now/
